packages/office-ui-fabric-react-experiments/package.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "@uifabric/experiments",
+  "version": "0.0.1",
+  "description": "Experimental React components for building experiences for Office 365.",
+  "main": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/OfficeDev/office-ui-fabric-react"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "gulp",
+    "clean": "gulp clean",
+    "start": "node node_modules/webpack-dev-server/bin/webpack-dev-server.js --config webpack.serve.config.js --open",
+    "test": "gulp"
+  },
+  "devDependencies": {
+    "@microsoft/load-themed-styles": "^1.5.1",
+    "@microsoft/web-library-build": "^3.2.1",
+    "@types/chai": "3.4.35",
+    "@types/enzyme": "2.8.0",
+    "@types/es6-promise": "^0.0.32",
+    "@types/mocha": "2.2.39",
+    "@types/prop-types": "15.5.1",
+    "@types/react": "15.0.38",
+    "@types/react-addons-test-utils": "0.14.18",
+    "@types/react-dom": "15.5.1",
+    "@types/resemblejs": "~1.3.28",
```

Comment:
It might be ok to leave them there, but it's better to start simple and add as needed. just a suggestion.